Gemfile

```diff
@@ -28,7 +28,7 @@ gem 'devise'
 
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.4.2', require: false
-gem 'pg', '~> 1.0'
```

Comment:
Just curious why you changed this?

Reply:
oh man....now i don't remember what was going on with that! That's why I have to remember to make more frequent commits..so sorry...I seem to remember I was having some trouble with postgres, maybe with devise? not sure...but I put the versioning back in and bundle installed and it all works as far as I can tell....